composer.json

```diff
@@ -22,9 +22,6 @@
         "ezsystems/ez-support-tools": "^0.1@dev",
         "knplabs/knp-menu-bundle": "^2.1"   
     },
-    "require-dev": {
```

Comment:
@mikadamczyk Why this PR removes phpunit? Seems unrelated/bad?

Reply:
Weird, actually when I reviewed this first it seemed like cleanup of redundant block, but maybe I misread something